libcodechecker/util.py

```diff
@@ -18,10 +18,10 @@
 
 import psutil
 
-from libcodechecker.logger import LoggerFactory
+from libcodechecker.logger import get_logger
 
 # WARNING! LOG should be only used in this module.
```

Comment:
This warning is only present in this file at this point. I think this can just as well be removed, we all see the pattern on how every module that logs contains a `LOG` global.

Reply:
Yes, we can remove it.